<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/tela_consultar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_consultar">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>590</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_retornar">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>590</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>RETORNAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_procura">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_reultado">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 217, 87);
border: solid;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Resultados&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_procura">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_reultado_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 217, 87);
border: solid;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Digite:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_resultados">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>1061</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(85, 255, 127);</string>
    </property>
    <column>
     <property name="text">
      <string>Código Aeronave</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data Validade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Operador</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPF/CNPJ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tipo Uso</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fabricante</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Número de Série</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cidade-Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ramo da Atividade</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
